tests/Integration/Mail/SendingMailWithLocaleTest.php

```diff
@@ -79,8 +79,8 @@ public function testMailIsSentWithLocaleUpdatedListenersCalled()
 
         Mail::to('test@mail.com')->locale('es')->send(new TimestampTestMail);
 
-        Assert::assertMatchesRegularExpression('/nombre (en|dentro de) (un|1) día/',
-            app('mailer')->getSwiftMailer()->getTransport()->messages()[0]->getBody()
+        Assert::assertMatchesRegularExpression('/nombre (en|dentro de) (un|1) d=C3=ADa/',
```

Comment:
You could use a custom cid name: https://symfony.com/doc/current/mailer.html#embedding-images
And return this one or let the Developer decide via an argument how the cid should be named.

Maybe defaulting to a random unique one?

Reply:
Short Answer:
Symfony Mailer calls `quoted_printable_encode('/nombre (en|dentro de) (un|1) día/')` which results in `'/nombre (en|dentro de) (un|1) d=C3=ADa/'`

Long Answer:
The TextPart of the Message will be converted in a string with the method `bodyToString()` (https://github.com/symfony/mime/blob/8b6fc5f7323a046dfee458169c62a3ec42294f56/Part/TextPart.php#L112-L115)

The body will then be encoded before returning. There are 3 possible encoders: (https://github.com/symfony/mime/blob/8b6fc5f7323a046dfee458169c62a3ec42294f56/Part/TextPart.php#L165-L176)
1. `quoted-printable`
2. `8bit`
3. `base64`

The default charset is `utf-8` and because of that the default encoder is `quoted-printable`(https://github.com/symfony/mime/blob/8b6fc5f7323a046dfee458169c62a3ec42294f56/Part/TextPart.php#L178-L185)

This Encoder will perform the encoding following the [RFC2045](http://www.faqs.org/rfcs/rfc2045.html), section 6.7.
which is implemented in the php function `quoted_printable_encode` (https://www.php.net/manual/en/function.quoted-printable-encode.php)
This function is also called from the Encoder itself. Therefor resulting in the different result than before. (See short answer)

This brings a great benefit for delivering the emails while also maintaining a good readability.

Reply:
Amazing. Thanks for figuring that out @Jubeki